dev-packages/application-manager/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@theia/application-manager",
+  "version": "0.3.7",
+  "description": "Theia application manager API.",
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/theia-ide/theia.git"
+  },
+  "bugs": {
+    "url": "https://github.com/theia-ide/theia/issues"
+  },
+  "homepage": "https://github.com/theia-ide/theia",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "prepare": "yarn run clean && yarn run build",
+    "clean": "theiaext clean",
+    "build": "theiaext build",
+    "watch": "theiaext watch",
+    "test": "theiaext test",
+    "docs": "theiaext docs"
+  },
+  "dependencies": {
+    "@types/fs-extra": "^4.0.2",
```

Comment:
Dependencies are the most heavier part.
Please check which dependencies are used in application-package and only them keep there. Others keep here.
Avoid duplication of dependencies. All dependencies available to the application package will be available to the manager.